ctapipe/image/tests/test_morphology.py

```diff
@@ -105,3 +105,22 @@ def test_largest_island():
     assert (mask_largest_one == true_mask_largest_one).all()
     assert (mask_largest_0 == true_mask_largest_0).all()
     assert_allclose(mask_largest, true_mask_largest)
+
+
+def test_brightest_island():
+    # we have two islands, 2 is larger but 1 is brighter
+    n_islands = 2
+    image = np.array([0, 1, 2, 5, 1, 0, 1, 2, 2, 1])
+    island_labels = np.array([0, 0, 1, 1, 1, 0, 2, 2, 2, 2])
```

Comment:
Shouldn't this be
`island_labels = np.array([0, 1, 1, 1, 1, 0, 2, 2, 2, 2])`

Reply:
No. It is an artificial test case. I wanted to have a smaller but brighter island. So island 1 has three pixels, island 2 has four but less photons.

Reply:
Ah so you are already considering a fictional image cleaning that kills the 2nd pixel?

Reply:
No, I just made a simple toy example. Don't interpret too much into the specific numbers